dashboard/test/models/plc/enrollment_unit_assignment_test.rb

```diff
@@ -3,37 +3,57 @@
 class Plc::EnrollmentUnitAssignmentTest < ActiveSupport::TestCase
   setup do
     @teacher = create :teacher
+    # It's easier to just use an existing course for this
```

Comment:
What's easier? Do you mean adding the course_unit to the existing course on line 8?